### PR TITLE
Add note about save_succeeded? -> saved? to upgrade notes

### PR DESCRIPTION
**Pull request by hanneskaeufler**
Sorry for the many tiny PRs.

### CODE REVIEW

File: UPGRADE_NOTES.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ UserForm.create!(name: "Jane")
 
 More info at https://luckyframework.org/guides/saving-with-forms/#passing-data-without-route-params
 
+* Change calls to `form.save_succeeded?` to `form.saved?`
```
Comment (edwardloveall) [inline UPGRADE_NOTES.md]: I'd suggest saying `from` first here. So its:

> Change calls **from** `form.save_succeeded?` to `form.saved?`

Comment (paulcsmith) [reply UPGRADE_NOTES.md]: I agree. I think that reads a bit better

**Comment by paulcsmith** [comment]
Thanks for these PRs. Small is great because they’re easy to review. So it’s no problem at all!

File: UPGRADE_NOTES.md (hunk 1 above)
Comment (hanneskaeufler) [reply UPGRADE_NOTES.md]: ✅

**Pull request merged**